### PR TITLE
doc: how to update comments.db

### PR DESCRIPTION
**Pull request by fliiiix**
Does this make sense enough? :sweat_smile:

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,3 +42,25 @@ stork --build stork.toml
...
+
+If you rename a post you should migrate the old comments.
+Since the id for comments is the post title: <https://github.com/Binaergewitter/serious-bg/blob/main/serious/lib/site/views/_isso.erb#L8>.
+
+1. Get the `comments.db`
+1. Create a backup
+1. Look up the old id and new id in the `threads` table (i use `sqlitebrowser`)
+1. Run the update command `UPDATE comments SET tid=$newid WHERE tid=$oldid;`
+1. Save file!
+1. Upload `comments.db` again
```
[the remark this answers is not in the document]
Comment (makefu) [reply README.md:56]: image

[the remark this answers is not in the document]
Comment (fliiiix) [reply README.md:56]: i don't make the markdown rules 
image

[the remark this answers is not in the document]
Comment (fliiiix) [reply README.md:56]: image


do i think it is kinda unhinged, yes

**Comment by rb2k** [comment]
Love it

**Pull request merged**